api/database/seeders/UserSeederUat.php

```diff
@@ -16,32 +16,187 @@ class UserSeederUat extends Seeder
      */
     public function run()
     {
-        $fakeEmailDomain = '@talent.test';
-
         // users with sub values from Sign In Canada, redirecting to uat-talentcloud.tbs-sct.gc.ca
-
         // Ensure we don't create duplicates if users already exist, since we don't truncate tables by default (unlike on local).
         User::updateOrCreate(
-            ['email' => 'tristan-orourke'.$fakeEmailDomain],
+            ['email' => 'usertester923@gmail.com'],
+            [
+                'first_name' => 'Wanda',
+                'last_name' => 'Maximoff',
+                'sub' => 'c0967038-f2bb-42cc-8bb3-4b90e9f77f89',
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['email' => "tristan.o'rourke@tbs-sct.gc.ca"],
             [
-                'first_name' => 'tristan-orourke',
-                'last_name' => 'Talent',
+                'first_name' => 'Tristan',
+                'last_name' => "O'Rourke",
                 'sub' => 'b4c734a1-dcf3-4fb8-a860-c642700cb0b8',
                 'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
             ]
         );
         User::updateOrCreate(
-            ['email' => 'petertgiles'.$fakeEmailDomain],
+            ['email' => 'petertgiles@gmail.com'],
```

Comment:
Speaking of changing emails... could you change mine to my work address. :-)  peter.giles@tbs-sct.gc.ca

Reply:
Yessir 😆 Changed in [054fa75](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5313/commits/054fa752aa439cbbee66d5354ce8a5d148e890a7)

Reply:
Thanks!